actions/form/class.EditClassLabel.php

```diff
@@ -113,7 +113,8 @@ protected function initElements()
             ]);
             $namespace = substr($clazz->getUri(), 0, strpos($clazz->getUri(), '#'));
             if (!$this->getNamespaceHelper()->isNamespaceSupported($namespace)) {
-                $readonly = \tao_helpers_form_FormFactory::getElement($element->getName(), 'Readonly');
+                $readonly = \tao_helpers_form_FormFactory::getElement('', 'Readonly');
```

Comment:
Could you put this in import block?
```suggestion
                $readonly = tao_helpers_form_FormFactory::getElement('', 'Readonly');
```

Reply:
`How is your migration fix duplicated label for test taker?`
it removes a duplicate, other changes in controllers does not allow to create them back